azext_iot/_help.py

```diff
@@ -505,14 +505,20 @@
     short-summary: Create an enrollment group in an Azure IoT Hub Device Provisioning Service.
     examples:
     - name: Create an enrollment group '[Enrollment ID]' in the Azure IoT provisioning service
-            '[DPS Name]' in the resource group '[Resource Group Name]'.
+            '[DPS Name]' in the resource group '[Resource Group Name] using an intermediate certificate as primary certificate'.
       text: >
         az iot dps enrollment-group create -g [Resource Group Name] --dps-name [DPS Name]
         --enrollment-id [Enrollment ID] --certificate-path /certificates/Certificate.pem
+    - name: Create an enrollment group '[Enrollment ID]' in the Azure IoT provisioning service
+            '[DPS Name]' in the resource group '[Resource Group Name] using a CA certificate [Certificate Name]
+            as secondary certificate'.
+      text: >
+        az iot dps enrollment-group create -g [Resource Group Name] --dps-name [DPS Name]
+        --enrollment-id [Enrollment ID] --secondary-certificate-name [Certificate Name]
     - name: Create an enrollment group '[Enrollment ID]' in the Azure IoT provisioning service
             'MyDps' in the resource group '[Resource Group Name]' with provisioning status
             'enabled', target IoT Hub '[IoTHub Name].azure-devices.net' and initial twin
```

Comment:
The '.azure-devices.net' part might be misleading as these commands should work across domains (for example the Chinese National Cloud).